projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -599,6 +599,8 @@ function wpcom_add_scheduled_updates_menu() {
 
 	$domain = wp_parse_url( home_url(), PHP_URL_HOST );
 
+	// TODO: When looking at wordpress.com/plugins/:site on an atomic site this menu
```

Comment:
This is the only thing that needs fixing I think 🤔

Reply:
Handled in Calypso: https://github.com/Automattic/wp-calypso/pull/91200